### DISCUSSION TITLE
مشکل توی کامپایل کردن سورس

### DISCUSSION THREAD

**Discussion by Amoiraj** [hiddify/hiddify-app#405] [Q&A]
سورس توی vscode اوردم ولی بعضی از فایلا نیست چیکار باید بکنم
![Screenshot 2024-02-01 210610](https://github.com/hiddify/hiddify-next/assets/156057575/8ef6847c-5f59-4f75-bd97-13738fca59c4)
اینجا هم میگه ازینا استفاده نشده و کلی ارور داره که پکیجاش نیست میشه کمک کنین
![Screenshot 2024-02-01 221232](https://github.com/hiddify/hiddify-next/assets/156057575/877a4b23-7c5f-4802-998f-fdb53f591637)

**Comment by LAziznezhad** [comment]
دقیقا منم این ارور رو دارم
